crop_mips.ijm

```diff
@@ -1,9 +1,29 @@
 
+macro "Crop Mips" {
 
+    chosen_folder = getDirectory("Select the animal ID or MIP folder");
 
-macro "Cell Count Crop [H]" {
+    if (endsWith(chosen_folder, "MIP\\")){
+        load_single(chosen_folder);
+    }
+    else{
+        load_multiple(chosen_folder);
+    }
+
+} // macro
+
+
+function crop(path, destination, x, y) {
+
+    makeRectangle(x, y, 1024, 1024);
```

Comment:
Do the (x,y) here correspond to the center of the rectangle or ?

Reply:
x, y correspond to the top left. (0, 0) is the top left of the image, and X, Y move positively right and down from there.